### ISSUE TITLE
Update extension name in settings

### ISSUE THREAD

**Issue by alexweininger** [microsoft/vscode-azureresourcegroups#447]
Since the extension display name is now "Azure Resources", for some (all?) settings, we should update the name there too. It still says "Azure Resource Groups" which is confusing.

<img width="529" alt="image" src="https://user-images.githubusercontent.com/12476526/206293568-ab13f7da-af73-49d5-95fa-acc6341f3120.png">

**Comment by alexweininger** [comment]
Looks like the only way to change that is by changing the setting id...

**Comment by bwateratmsft** [comment]
> Looks like the only way to change that is by changing the setting id...

That's nearly a nonstarter ☹️

**Comment by alexweininger** [comment]
> > Looks like the only way to change that is by changing the setting id...
> 
> That's nearly a nonstarter ☹️

Yeah, we need help from VS Code on this one. I think the simplest option would be to let us control the string that gets used to construct the setting title. I was surprised we aren't able to.

**Comment by bwateratmsft** [comment]
It's possible to migrate settings, but I wouldn't feel good about doing so at this scale.

**Comment by alexweininger** [comment]
Opened a discussion https://github.com/microsoft/vscode-discussions/discussions/406

**Comment by alexweininger** [comment]
Tracking https://github.com/microsoft/vscode/issues/150891
